### DISCUSSION TITLE
Gatsby Develop can't establish connection to localhost:8000 (CORS issue?)

### DISCUSSION THREAD

**Discussion by Svarto** [gatsbyjs/gatsby#28857] [Help]
Hi all,

I am trying to get up and running with Gatsby start package (https://github.com/gillkyle/gatsby-starter-landing-page), but running into problems in Firefox where I see errors such as "can't read hot-load.json, can't establish connection to /socket.io and a continuous stream of CORS errors.

I have likely done something wrong, but tried following the tutorial and have searched documents and issues + discussion but can't find anything that seems related to this (except a more advanced error around WSL2, which I don't think is related to my case).

I tried to also run `gatsby develop -H 0.0.0.0`, but this didn't fix or change anything unfortunately.

Does anyone have any idea or suggestion how I can approach fixing these errors: 

![image](https://user-images.githubusercontent.com/24279217/103632344-10989800-4f3c-11eb-8fb5-d18aec64eb22.png)

Thank you for your help!

**Comment by marcel0ll** [comment]
Hey @Svarto could you please check your package-lock.json or yarn-lock.json for different socket.io versions? I suspect that this is the actual problem. In my case, it looks like gatsby is using 2.3.0 and something else was using 2.4.0

The only thing that seems to have solved it was to use yarn resolutions and set `"socket.io": "2.3.0"

**Comment by marcel0ll** [comment]
Hey @Svarto this should be solved, Gatsby locked the version to 2.3.0 for now. Have a look [here](https://github.com/gatsbyjs/gatsby/issues/28877)
